src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -468,4 +471,64 @@ object Task {
   }
 
   implicit val taskFormat: Format[Task] = Json.format[Task]
+
+  def provisioned(taskId: Id, networkInfo: NetworkInfo, appVersion: Timestamp, now: Timestamp): Task = {
```

Comment:
this was originally part of Instance class, but it's actually creating tasks so I moved it here. I think it's a better fit

Reply:
Ah, I was just wondering why it is not part of the task class. Could you add some docs?

Reply:
As discussed it would be nice to unify the `Task.provisioned` for apps and pods in a follow up.

Reply:
MARATHON-8521